breast_basal/README.md

```diff
@@ -0,0 +1,146 @@
+
+Cancer Systems Biology, Section of Bioinformatics, Department of Health and Technology, Technical University of Denmark, 2800, Lyngby, Copenhagen
+
+# Moonlight2_DMA_basal_like
+
+An Automatized Workflow to Study Mechanistic Indicators for Driver Gene Prediction with Moonlight
+Astrid Saksager, Mona Nourbakhsh, Nikola Tom, Xi Steven Chen, Antonio Colaprico, Catharina Olsen, Matteo Tiberti, Elena Papaleo*, submitted to biorxiv
+
+contacts for the repository: Elena Papaleo, elpap-at-dtu.dk, elenap-at-cancer.dk; Astrid Saksager: abrisa-at-dtu.dk
+
+PLEASE, CITE THE PUBLICATION ABOVE IF YOU USE THE SCRIPTS FOR YOUR OWN RESEARCH
+
+This Project-folder contains the entire Basal-like Breast Cancer study
+AIM: To find driver genes for Basal-like patients by running Moonlight2 on BRCA-Basal specific data from TCGA.
+
+## Installing requirements and reproducing the analysis
+
+All the analyses have been performed on a GNU/Linux server.
+
+NB: When reproducing the analyses and results, the user cannot expect to obtain identical results to the ones
+in this GitHub repository and associated with the publication due to stochastic processes in the GRN step of the Moonlight protocol. 
+
+### CScape-somatic
+
+In order to run the analyses you will need to have available the pre-calculated
+[CScape-somatic](http://cscape-somatic.biocompute.org.uk) scores. These are
+downloadable from CScape-somatic website. Please download the `css_coding.vcf.gz`,
+`css_noncoding.vcf.gz`, `css_coding.vcf.gz.tbi` and `css_noncoding.vcf.gz.tbi`
+files and store them or link them in the main folder of the cloned repository
+(see step 2 in the following section).
+
+The `Run_Basal.sh` script can be optionally modified to set a custom location of
+the CScape file. **If this is a relative path, it should start from the `scripts`
+directory inside the main folder, since that is where the code is run from.**
+
+### Computing environment
+
+In order to reproduce the paper data, you will need to set up a conda environment
+on which the expected version of R and the required packages will be installed;
+this requires being able to run Anaconda by means of the `conda` executable.
+
+If you don't have access to `conda` please see the [Miniconda installer page](https://docs.conda.io/en/latest/miniconda.html) for instructions on how to install Miniconda.
+
+Once you have access to `conda`, you can
+
+1. clone our github repository into a local directory on your local machine:
+
+```
+git clone https://github.com/ELELAB/Moonlight2_DMA_basal_like.git
+cd Moonlight2_DMA_basal_like
+```
```

Comment:
I think we need to remove from here all the steps we cited in the general README.md file. The idea would be:

- user reads general README.md file and prepare the environment that will work for all case studies
- user cds into one of the folders and reads the README.md file in there to prepare what else the specific case study needs

this should also be mentioned in this file (in order to prepare the virtual environment for this case study, please see the README file in the main folder...)